src/pyinit.jl

```diff
@@ -114,4 +115,7 @@ function __init__()
             end
         end
     end
+
+    # Will get reinitialized properly on first use
+    is_windows() && (PyActCtx[] = C_NULL)
```

Comment:
It seems like this has to go at the top of `__init__`, not at the bottom.   (Note that `__init__` calls `pyimport` several times.)